### PR TITLE
Issue #SB-19635 fix: Added reCaptcha validation for user exists API

### PR DESCRIPTION
**Pull request by rajeshkumaravel**
- Issue #SB-19635 fix: Added reCaptcha validation for user exists API

### CODE REVIEW

**Comment by rajeshkumaravel** [comment]
image

File: src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts [hunk 1]
```diff
@@ -186,7 +188,7 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
     });
     this.onContactTypeValueChanges();
     this.enableSignUpSubmitButton();
-    this.onPhoneChange();
+    // this.onPhoneChange();
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts:191]: can you please remove this line as this is no longer needed

File: src/app/routes/learnerRoutes.js [hunk 2]
```diff
@@ -43,6 +44,15 @@ module.exports = function (app) {
     healthService.checkDependantServiceHealth(['LEARNER', 'CASSANDRA']),
     proxyManagedUserRequest()
   )
+
+  app.get('/learner/user/v1/exists/email/:emailId',
+    googleService.validateRecaptcha
+  );
+
+  app.get('/learner/user/v1/exists/phone/:phoneNumber',
```
Comment (rajeevsathish) [inline src/app/routes/learnerRoutes.js:52]: what is the further process of this post the validateRecaptcha?

Comment (rajeshkumaravel) [reply src/app/routes/learnerRoutes.js:52]: - Route wildcards will handle request forward

File: src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts (hunk 1 above)
Comment (rajeshkumaravel) [reply src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts]: 👍

**Pull request merged**